build.gradle

```diff
@@ -142,25 +132,19 @@ configurations {
     cucumberRuntime.extendsFrom(functionalRuntime)
 }
 
-checkstyle {
-    showViolations = true
-    maxWarnings = 0
-    toolVersion = '8.18'
-}
+//checkstyleMain {
```

Comment:
should this be here?

Reply:
Well, it looks like it's just missed.
@skandala-cgi, could you please remove checkstyle plugin traces entirely?